org.eclipse.xtext.doc/contents/350_continuous_integration.html

```diff
@@ -254,6 +254,54 @@ <h2 id="maven-tycho-hints">Maven Tycho Hints</h2>
     </tr>
   </thead>
   <tbody>
+    <tr>
+      <td><a href="http://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.17.0/">2.17.0</a></td>
+      <td><a href="http://download.eclipse.org/releases/2019-03">2.16.0</a> (2.12.0)</td>
+      <td><a href="http://download.eclipse.org/modeling/emft/mwe/updates/releases/2.9.0/">2.9.0</a> (2.7.1)</td>
+      <td><a href="http://download.eclipse.org/modeling/m2t/xpand/updates/releases/R201605260315">2.2.0</a> (1.4)</td>
+      <td><a href="http://download.eclipse.org/releases/2019-03">4.11.0</a> (4.7.3)</td>
```

Comment:
Why is this the same URL?

Reply:
All the features are on the SimRel repository. No need to use more repos.